examples/run/cpu/full_chain_algorithm.hpp

```diff
@@ -62,6 +65,11 @@ class full_chain_algorithm
     /// Track parameter estimation algorithm
     track_params_estimation m_track_parameter_estimation;
 
+    /// Configs
+    seedfinder_config m_finder_config;
+    spacepoint_grid_config m_grid_config;
+    seedfilter_config m_filter_config;
```

Comment:
:frowning: Why...?

Much like how the different `seeding_algorithm`-s didn't need to hold on to any configuration objects themselves, the `full_chain_algorithm`-s don't need to do it either. They just need to pass these on to the "sub-algorithms" in their constructors, and that's it. No new members should be added to these classes.

Reply:
Otherwise, I cannot pass the config parameters in the copy constructor. Do you have an idea?

```
      m_seeding(
          parent.m_finder_config, parent.m_grid_config, parent.m_filter_config,
          memory_resource{*m_cached_device_mr, &m_host_mr}, m_copy, m_stream),
```

Reply:
Ahh, crud... 😦 

The correct way of doing this would of course be to implement proper copy-constructors for all of those sub-algorithms. And have `full_chain_algorithm` just rely on those copy-constructors.

But fine, that's a fair amount of extra code that this PR would have to collect. Okay, let's just go with this for now then, we can clean up the code a little more in a future PR.